src/presentation/public/manifest.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "privacy.sexy",
+    "short_name": "privacy.sexy",
+    "description": "Enforce privacy & security best-practices on Windows, macOS and Linux, because privacy is sexy.",
+    "version": "0.13.5",
+    "start_url": "/",
+    "display": "standalone",
+    "background_color": "#ffffff",
+    "theme_color": "#000000",
+    "icons": [
+        {
+          "src": "icons/manifest-icon-192.maskable.png",
+          "sizes": "192x192",
+          "type": "image/png",
+          "purpose": "any"
+        },
+        {
+          "src": "icons/manifest-icon-192.maskable.png",
+          "sizes": "192x192",
+          "type": "image/png",
+          "purpose": "maskable"
+        },
+        {
+          "src": "icons/manifest-icon-512.maskable.png",
+          "sizes": "512x512",
+          "type": "image/png",
+          "purpose": "any"
+        },
+        {
+          "src": "icons/manifest-icon-512.maskable.png",
+          "sizes": "512x512",
+          "type": "image/png",
+          "purpose": "maskable"
+        }
+      ],
+      "screenshots": [
+        {
+          "src": "screenshots/screenshot-1280x720.png",
+          "sizes": "1280x720",
+          "type": "image/png",
+          "form_factor": "wide" 
+        },
+        {
+            "src": "screenshots/screenshot-1366x768.png",
+            "sizes": "1366x768",
+            "type": "image/png",
+            "form_factor": "wide" 
+        },
+        {
+            "src": "screenshots/screenshot-1600x900.png",
+            "sizes": "1600x900",
+            "type": "image/png",
+            "form_factor": "wide" 
+        },
+        {
+            "src": "screenshots/screenshot-1920x1080.png",
+            "sizes": "1920x1080",
+            "type": "image/png",
+            "form_factor": "wide" 
+        }
+      ]
```

Comment:
Do we need those? If we do, it would be hard to maintain screenshots images on each version change. Can we create like a conceptual picture like this without really showing the application but a concept with cards and code section so it can be reused all the time? Or is the point of screenshot to be real screenshots?

![image](https://github.com/user-attachments/assets/59121561-91e8-4fc0-9446-c8029a7de758)

Reply:
TLDR;

https://web.dev/articles/add-manifest#screenshots

Reply:
We can also use [`img/screenshot.png`](https://github.com/undergroundwires/privacy.sexy/blob/master/img/screenshot.png). We can move the file to public directory, and change its references to the public directory. It would be single file but easier to maintain.

Reply:
We actually need screenshots of every sizes

Reply:
Another option is to use `puppeteer` to automate the flow. We can create a script that will build the application, load in a web browsers, take screenshots of different sizes and stores them.

Or else, we can just remove the screenshots from this PR and think / solve in future.

Reply:
Ok will see 🫡